regexes/device/car_browsers.yml

```diff
@@ -7,6 +7,10 @@
 
 # Tesla Model S
 Tesla:
-  regex: 'QtCarBrowser'
+  regex: '(?:Tesla/[0-9.]+|QtCarBrowser)'
   device: 'car browser'
-  model: 'Model S'
+  models:
+    - regex: 'QtCarBrowser'
+      model: 'Model S'
+    - regex: 'Tesla/[0-9.]+'
+      model: Tesla Vehicle
```

Comment:
Maybe we should leave the name empty in that case as it's actually no specific model

Reply:
I'm good with that. I thought about putting the firmware version there, but that didn't really make sense to me either.